build/flac-1.3.1/src/libFLAC/libFLAC_static.vcxproj

```diff
@@ -96,6 +96,10 @@
       <ProgramDataBaseFileName>$(OutDir)$(TargetName).pdb</ProgramDataBaseFileName>
       <RuntimeLibrary>MultiThreadedDebugDLL</RuntimeLibrary>
     </ClCompile>
+    <Lib>
```

Comment:
What's confusing is with the right project dependency settings, you're not supposed to have to do this.